rust/src/http2/http2.rs

```diff
@@ -58,8 +59,8 @@ const HTTP2_FRAME_GOAWAY_LEN: usize = 4;
 const HTTP2_FRAME_RSTSTREAM_LEN: usize = 4;
 const HTTP2_FRAME_PRIORITY_LEN: usize = 5;
 const HTTP2_FRAME_WINDOWUPDATE_LEN: usize = 4;
-//TODO make this configurable
-pub const HTTP2_MAX_TABLESIZE: u32 = 0x10000; // 65536
+pub static mut HTTP2_MAX_TABLESIZE: u32 = 0x10000; // 65536
```

Comment:
I prefer these to be in human readable format. To the compiler its all the same and it avoids the need to add comments like here

Reply:
Ok.
But turning it into `65536; // 0x10000`
To me, the human readable version is where we see that we have a power of two